app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -622,7 +622,25 @@ class NewMessageViewModel @Inject constructor(
     private fun isSavingDraftWithoutChanges(action: DraftAction) = action == DraftAction.SAVE && snapshot?.hasChanges() != true
 
     fun updateIsSendingAllowed() {
-        isSendingAllowed.value = draft.to.isNotEmpty() || draft.cc.isNotEmpty() || draft.bcc.isNotEmpty()
+
+        val hasRecipient = draft.to.isNotEmpty() || draft.cc.isNotEmpty() || draft.bcc.isNotEmpty()
```

Comment:
Could be great to have a method in the `Draft` class to do this check? `draft.hasRecipient()` ?